src/tls/test/main.cpp

```diff
@@ -375,6 +396,36 @@ TEST_CASE("large message")
   auto server_cert = get_dummy_cert(ca, "server");
   auto client_cert = get_dummy_cert(ca, "client");
 
+  LOG_INFO_FMT("large message");
+
+  // Testing communication channel, verifying certificates.
+  run_test_case(
+    0,
+    (const uint8_t*)message.data(),
+    message.size(),
+    (const uint8_t*)message.data(),
+    message.size(),
+    move(server_cert),
+    move(client_cert),
+    true);
+}
+
+TEST_CASE("very large message")
+{
+  // Uninitialised on purpose, we don't care what's in here
+  size_t len = 16149;
```

Comment:
That's less than 16k?

Reply:
yes, but the base64 will be more.

Reply:
though, I could just set that to 16k...

Reply:
Ah indeed, still kind of a weird number to pick but ok.